.travis.yml

```diff
@@ -3,12 +3,13 @@ notifications:
 sudo: required
 services:
   - docker
+dist: xenial
```

Comment:
This is required for Python 3.7 and higher: https://docs.travis-ci.com/user/languages/python/#python-37-and-higher